### PR TITLE
Added styles larger footer.

### PR DESCRIPTION
**Pull request by kanastasakis26**
Tweaked layout of footer in VIPER web
Added styles to text elements in the footer. 
Added a supporting mmui-footer style that allows the footer to grow vertically. This should be helpful when more items are added to the footer.

### CODE REVIEW

File: src/sass/mmui-styles/mmui_theme.scss [hunk 1]
```diff
@@ -32,8 +32,30 @@ html{
...
+    font-family: $font-stack-sans-serif;
+    font-size: 1.4rem;
+    color: $mm-white;
+    margin-bottom: 5px;
+  }
 }
 
+.mmui-footer-tall{
+  height: auto;
+  min-height: 70px;
```
Comment (besseym) [inline src/sass/mmui-styles/mmui_theme.scss]: Do you need the `min-height` on this `mmui-footer-tall` class as well as the `container-fluid` child?

File: src/sass/mmui-styles/mmui_theme.scss [hunk 2]
```diff
@@ -32,8 +32,30 @@ html{
...
+  }
+  .footer-list-header{
+    font-family: $font-stack-sans-serif;
+    font-size: 1.4rem;
+    color: $mm-white;
+    margin-bottom: 5px;
+  }
 }
 
+.mmui-footer-tall{
```
Comment (besseym) [inline src/sass/mmui-styles/mmui_theme.scss]: Does this need to be a separate class or can we this be our new default footer height on `mmui-footer`?

Comment (kanastasakis26) [reply src/sass/mmui-styles/mmui_theme.scss]: If we are to strictly follow the design, then yes.

File: src/sass/mmui-styles/mmui_theme.scss (hunk 1 above)
Comment (kanastasakis26) [reply src/sass/mmui-styles/mmui_theme.scss]: The min hieght in the design is 70px, using the min-height guarantees that.

Comment (kanastasakis26) [reply src/sass/mmui-styles/mmui_theme.scss]: https://zpl.io/amDmEEm

File: src/sass/mmui-styles/mmui_theme.scss (hunk 2 above)
Comment (kanastasakis26) [reply src/sass/mmui-styles/mmui_theme.scss]: https://zpl.io/amDmEEm

File: src/sass/mmui-styles/mmui_theme.scss (hunk 1 above)
Comment (besseym) [reply src/sass/mmui-styles/mmui_theme.scss]: Yes, but does `min-height: 70px;` need to be defined twice in both `.mmui-footer-tall` and `.mmui-footer-tall .container-fluid`

File: src/sass/mmui-styles/mmui_theme.scss (hunk 2 above)
Comment (besseym) [reply src/sass/mmui-styles/mmui_theme.scss]: I'd like to check with Stephanine on if we can make the styles in `mmui-footer-tall` the new standard for `mmui-footer`.

File: src/sass/mmui-styles/mmui_theme.scss (hunk 1 above)
Comment (kanastasakis26) [reply src/sass/mmui-styles/mmui_theme.scss]: It is defined twice in mmui-footer. I assumed it was done for a reason. 
I copied the original structure.

File: src/sass/mmui-styles/mmui_theme.scss (hunk 2 above)
Comment (kanastasakis26) [reply src/sass/mmui-styles/mmui_theme.scss]: Sure. The only problem is that the footer will default to a height of 80px. I did not investigate why, just noticed while developing. 
We should try to get this done before Tuesday.

**Pull request merged**